android/KMEA/app/src/main/java/com/tavultesoft/kmea/ModelPickerActivity.java

```diff
@@ -82,10 +82,10 @@ protected void onResume() {
     super.onResume();
     if (!didExecuteParser) {
       didExecuteParser = true;
-      //new JSONParse().execute();
       repo = CloudRepository.shared.fetchDataset(context);
 
-      listView.setAdapter(new FilteredLexicalModelAdapter(context, repo, languageID));
+      // Initialize the dataset of installed lexical models
+      listView.setAdapter(new FilteredLexicalModelAdapter(context, KeyboardPickerActivity.getInstalledDataset(context), languageID));
```

Comment:
This change needed so Model Info page will correctly show the installed versions.